src/main/java/io/xpring/javascript/JavaScriptUtils.java

```diff
@@ -31,6 +32,41 @@ public boolean isValidAddress(String address) {
         return isValidAddressFunction.execute(address).asBoolean();
     }
 
+    /**
+     * Encode the given classic address and tag into an X-Address.
+     *
+     * @param classicAddress A classic address to encode.
+     * @param tag An optional tag to encode.
+     * @return A new X-Address if inputs were valid, otherwise null.
+     * @see <a href="https://xrpaddress.info/">https://xrpaddress.info/</a>
+     */
+    public String encodeXAddress(String classicAddress, Long tag) {
```

Comment:
Is there a reason to not accept an instance of `ClassicAddress` as the argument here instead of its decomposed fields?